skopt/space.py

```diff
@@ -255,7 +258,8 @@ def rvs(self, n_samples=None, random_state=None):
 
     @property
     def transformed_size(self):
-        return len(self.categories)
+        l = len(self.categories)
+        return l if l != 2 else 1
```

Comment:
`l` -> `size` maybe? Right now it looks a bit like `return 1 if 1 != 2 else 1` :)